clients/src/main/java/org/apache/kafka/common/config/SslConfigs.java

```diff
@@ -108,6 +108,9 @@
 
     public static final String SSL_SECURE_RANDOM_IMPLEMENTATION_CONFIG = "ssl.secure.random.implementation";
     public static final String SSL_SECURE_RANDOM_IMPLEMENTATION_DOC = "The SecureRandom PRNG implementation to use for SSL cryptography operations. ";
+	
+	public static final String SSL_APPKI_PROVIDER_CLASS_CONFIG = "ssl.appki.provider.class";
```

Comment:
You should fix formatting so these line up correctly.

Reply:
No idea why it shows up this way here, the idea editor shows proper formatting